stack.yaml.lock

```diff
@@ -0,0 +1,12 @@
+# This file was autogenerated by Stack.
```

Comment:
Don't think you should commit this file?

Reply:
I'm a bit confused. That's the whole file. Should I remove it and or add it to `gitignore`?

Reply:
I don't have any idea about what this file does but @jneira advided today that we should not commit them.

Reply:
Sounds good, I'll add it to `gitignore`